app/models/comment.rb

```diff
@@ -1,4 +1,9 @@
 class Comment < ApplicationRecord
   belongs_to :user
```

Comment:
[REQUIRED] This should be ``belongs_to :author, class_name: 'User'``.It helps. to differentiate between the different types of users.